selfcheckout.py

```diff
@@ -54,9 +54,15 @@ def loan(userid, barcode):
     #del params['item_barcode']
     loans_response = requests.get(url, params=params)
     already_checked_out = loans_response.json().get('item_loan', False)
+    #error handling
     if already_checked_out:
-        return Response('Already Checked Out', 409)
-    
+        return Response('This item is already checked out', 409)
+    if loans_response.status_code == 404:
+    	return Response('Error incorrect barcode', 409)
```

Comment:
This one can't be a 404, if you put a slash in the barcode it will return a strange error code about going to an html page.